cmds/core/sluinit/uinit.go

```diff
@@ -110,3 +114,68 @@ func unmountAndExit() {
 	time.Sleep(5 * time.Second) // let queued up debug statements get printed
 	os.Exit(1)
 }
+
+// scanIscsiDrives calls iscsinl to mount iscsi drives.
+// format: netroot=iscsi:@X.Y.Z.W::3260::iqn.FOO.com.abc:hostname-boot
+func scanIscsiDrives() error {
+
+	slaunch.Debug("Scanning kernel cmd line for *netroot* flag")
+	val, ok := cmdline.Flag("netroot")
+	if !ok {
+		return errors.New("netroot flag is not set")
+	}
+
+	// val = iscsi:@10.196.210.62::3260::iqn.1986-03.com.sun:ovs112-boot
```

Comment:
Per our discussion before, it is likely better to say this is the only supported format.